index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Porfolio</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+        <header>
+
+            <nav>
+        
+                <ul class="navbar">
+                    <li>
+                    <a href="#"> <img src="images/logo.svg" alt="logo"> </a>
+                    </li>
+                    <li>
+                    <a href="#"> <img src="images/menu.svg" alt="menu"> </a>
+                    </li>
+
+                </ul>
+            </nav>
+
+        </header>
+
+        <section>
+            <div class="container">
+                <div class="text">
+                    <h1>I’m Mariane Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div class="connect">
+                    <p> LET'S CONNECT</p>
+                    <div class="logo"> 
+                        <a href="#"><img src="images/twitter.svg"> </a>
+                        <a href="#"><img src="images/linkedin.svg"> </a>
+                        <a href="#"><img src="images/medium.svg"> </a>
+                        <a href="#"><img src="images/github.svg"> </a>
+                        <a href="#"><img src="images/peace.svg"> </a>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
Please use flexbox for the layout of this section, This gives us a lot of options to align all our elements, plus, it's responsive.
also the project requirements states _**"In order to lay out the elements on the page you should use Flexbox in all 2 sections."**_ in order for you to be comfortable with flex and [become a flex ninja](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), which is a crucial skill in front-end development.

Reply:
Thank you For the feedback. I really appreciate that.